README.md

```diff
@@ -2,3 +2,39 @@
 Gym for predictive models
 
 [![run at everware](https://img.shields.io/badge/run%20me-@everware-blue.svg?style=flat)](https://everware.ysda.yandex.net/hub/oauth_login?repourl=https://github.com/yandexdataschool/modelgym)
+
+## model gym with Docker
+
+1. [Getting started](#1-getting-started)  
+2. [Running model gym in Docker container](#2-running-model-gym-in-docker-container)
+
+### 1. Getting started
+To run model gym inside Docker container you need to have installed
+[Docker](https://docs.docker.com/engine/installation/#supported-platforms) (also for Mac and Windows you can install instead [Kitematic](https://kitematic.com)). 
+
+Download this repo. All of the needed files are in the `modelgym` directory.
+```sh
+$ git clone https://github.com/yandexdataschool/modelgym.git
+$ cd ./modelgym
+```
+
+### 2. Running model gym in Docker container
+In order to run model gym in a single docker containter you can pull model gym Docker image `anaderi/modelgym:0.1.3` from official [repo](https://hub.docker.com/r/anaderi/modelgym/) on DockerHub.
```

Comment:
I'd not mention specific version, as it might change.. use `anaderi/modelgym:latest` instead.